Tests/UnitTests/Networking/Responses/OfferingsDecodingTests.swift

```diff
@@ -48,13 +49,51 @@ class OfferingsDecodingTests: BaseHTTPResponseTest {
     }
 
     func testDecodesSecondOffering() throws {
-        let offering = try XCTUnwrap(self.response.offerings.last)
+        let offering = try XCTUnwrap(self.response.offerings[safe: 1])
 
         expect(offering.identifier) == "alternate"
         expect(offering.description) == "alternate offering"
+        expect(offering.metadata) == [:]
         expect(offering.packages).to(haveCount(1))
 
-        let package = offering.packages[0]
+        let package = try XCTUnwrap(offering.packages[safe: 0])
+
+        expect(package.identifier) == PackageType.lifetime.description
+        expect(package.platformProductIdentifier) == "com.revenuecat.other_product"
+    }
+
+    func testDecodesMetadataOffering() throws {
+        let offering = try XCTUnwrap(self.response.offerings[safe: 2])
+
+        expect(offering.identifier) == "metadata"
+        expect(offering.description) == "offering with metadata"
+        expect(offering.metadata) == [
+            "int": 5,
+            "double": 5.5,
+            "boolean": true,
+            "string": "five",
+            "array": ["five"],
+            "dictionary": [
+                "string": "five"
+            ]
+        ]
+        expect(offering.packages).to(haveCount(1))
+
+        let package = try XCTUnwrap(offering.packages[safe: 0])
+
+        expect(package.identifier) == PackageType.lifetime.description
+        expect(package.platformProductIdentifier) == "com.revenuecat.other_product"
+    }
+
+    func testDecodesNullMetadataOffering() throws {
+        let offering = try XCTUnwrap(self.response.offerings[safe: 3])
+
+        expect(offering.identifier) == "nullmetadata"
+        expect(offering.description) == "offering with null metadata"
+        expect(offering.metadata) == [:]
+        expect(offering.packages).to(haveCount(1))
+
+        let package = try XCTUnwrap(offering.packages[safe: 0])
```

Comment:
I'd use `first` here which is the same:
```suggestion
        let package = try XCTUnwrap(offering.packages.first)
```